File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/login/LoginView.java
package ru.coffeeplanter.androidlogin.presentation.fragments.login;

import ru.coffeeplanter.androidlogin.presentation.BaseView;

@SuppressWarnings("SpellCheckingInspection")
interface LoginView extends BaseView {

    void switchToWaitingMode();
    void switchOffWaitingMode();
    void setLoginError(String error);
    void setPasswordError(String error);
    void clearFieldsData();
    void setLoginFocused();
    void setPasswordFocused();
    void setFocusedView();
    void navigateToLoggedInFragment(String login, String password);
    void showKeyboard();
    void hideKeyboard();
    void fadeStatusBar();
    void unfadeStatusBar();

}

File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/login/LoginView.kt
package ru.coffeeplanter.androidlogin.presentation.fragments.login

import ru.coffeeplanter.androidlogin.presentation.BaseView

internal interface LoginView : BaseView {

    fun switchToWaitingMode()
    fun switchOffWaitingMode()
    fun setLoginError(error: String?)
    fun setPasswordError(error: String?)
    fun clearFieldsData()
    fun setLoginFocused()
    fun setPasswordFocused()
    fun setFocusedView()
    fun navigateToLoggedInFragment(login: String, password: String)
    fun showKeyboard()
    fun hideKeyboard()
    fun fadeStatusBar()
    fun unfadeStatusBar()

}
